logic/src/jvmMain/kotlin/com/wire/kalium/logic/configuration/ClientConfigImpl.kt

```diff
@@ -18,6 +18,6 @@ actual class ClientConfigImpl : ClientConfig {
     }
 
     override fun clientType(): ClientType {
-        return ClientType.Temporary
+        return ClientType.Permanent
     }
```

Comment:
@typfel does changing this break something for MLS and the like?

Reply:
The effect is that CLI will register permanent clients. Not end of the world but we should make this configurable.

Reply:
Has no impact on MLS

Reply:
I can make a PR for configuration of ClientType once this PR is merged

Reply:
I reverted the change so that this PR can be merged without affecting CLI